astropy/time/formats.py

```diff
@@ -14,7 +14,8 @@
 from astropy.utils.decorators import classproperty, lazyproperty
 from astropy.utils.exceptions import AstropyDeprecationWarning
 
-from . import _parse_times, conf, utils
```

Comment:
Is this change necessary or even desired?

Reply:
moving ``from . import conf`` to ``from .config import conf``?